hugo/layouts/partials/create_description.html

```diff
@@ -0,0 +1,59 @@
+<!-- creates the site meta description for each page, good for search engine optimization -->
+
+{{ $description := "" }}
+
+{{ if .IsHome }}
+    {{ $description = "A platform facilitating access and discovery of genome data of non-model eukaryotic species studied in Sweden, hosted by SciLifeLab" }}
+
+<!-- Individ pages -->
```

Comment:
Would it be useful for the SEO if each individual page description mentions "The Swedish Reference Portal" by full name? Many of them already do, but not all. Or is it enough if only Home does that? (Perhaps it would be a form of keyword spamming?)

Reply:
I'm honestly not sure, it could be that we are over- or under-using "Swedish Reference Genome Portal". The examples I've seen on the internet don't often seem to mention the website name/title in the description so I suspect it is not necessary/essential at least.